endpoints/authentication/src/main.cpp

```diff
@@ -42,8 +42,9 @@
 #include <vector>
 
 // clang-format off
-namespace beast = boost::beast; // from <boost/beast.hpp>
-namespace net   = boost::asio;  // from <boost/asio.hpp>
+namespace beast   = boost::beast; // from <boost/beast.hpp>
+namespace net     = boost::asio;  // from <boost/asio.hpp>
+namespace logging = irods::http::log;
```

Comment:
Should this be "logger" or "logging"? Just going off the commit message here.

Reply:
Oops. That should be `logging` _(since that is what's used throughout the endpoint impls)_.

Nice catch.

Reply:
Fixed commit message.